### ISSUE TITLE
[@types/jquery@3.2.18] Type '"msCaching"' is not assignable to type...

### ISSUE THREAD

**Issue by jpoehls** [DefinitelyTyped/DefinitelyTyped#28000]
If you know how to fix the issue, make a pull request instead.

- [x] I tried using the `@types/jquery` package and had problems.
- [x] I tried using the latest stable version of tsc. https://www.npmjs.com/package/typescript
- [x] I have a question that is inappropriate for [StackOverflow](https://stackoverflow.com/).  (Please ask any appropriate questions there).
- [x] [Mention](https://github.com/blog/821-mention-somebody-they-re-notified) the authors (see `Definitions by:` in `index.d.ts`) so they can respond.
  - Authors: @leonard-thieu, @borisyankov, @Steve-Fenton, @Diullei 

--------------

When using the `@types/jquery@3.2.18` package I get the following compilation error under TypeScript 3.0.1:

```
node_modules/@types/jquery/index.d.ts:6126:66 - error TS2344: Type '"timeout" | "onreadystatechange" | "responseType" | "withCredentials" | "msCaching"' does not satisfy the constraint '"abort" | "open" | "timeout" | "getAllResponseHeaders" | "getResponseHeader" | "overrideMimeType" | "readyState" | "responseText" | "setRequestHeader" | "status" | "statusText" | ... 22 more ... | "dispatchEvent"'.
  Type '"msCaching"' is not assignable to type '"abort" | "open" | "timeout" | "getAllResponseHeaders" | "getResponseHeader" | "overrideMimeType" | "readyState" | "responseText" | "setRequestHeader" | "status" | "statusText" | ... 22 more ... | "dispatchEvent"'.

6126         interface XHRFields extends Partial<Pick<XMLHttpRequest, 'onreadystatechange' | 'responseType' | 'timeout' | 'withCredentials' | 'msCaching'>> { }
```

Removing `msCaching` from the union on line 6126 fixes the build error but I'm not familiar enough with the jQuery types to know whether this is the best way to resolve this issue.

**Comment by leonard-thieu** [comment]
This is fixed by #26524. Please update to a recent version of `@types/jquery`.

**Comment by orta** [comment]
Hi thread, we're moving DefinitelyTyped to use [GitHub Discussions](https://github.com/DefinitelyTyped/DefinitelyTyped/issues/53377) for conversations the `@types` modules in DefinitelyTyped.

To help with the transition, we're closing all issues which haven't had activity in the last 6 months, which includes this issue. If you think closing this issue is a mistake, please pop into the [TypeScript Community Discord](https://discord.gg/typescript) and mention the issue in the `definitely-typed` channel.
